priya/javascript/Assignment11/JsAssign11Q1.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Roll number and Marks</title>
+  </head>
+  <body>
+    Enter number: <input type="text" id="pic">
+    <button onclick="show_roll()">show</button>
+    <p id="demo"></p>
+    <script>
+      function show_roll() {
+        var p;
+        p = parseInt(document.getElementById("pic").value);
+        document.getElementById("demo").innerHTML = `roll no :${p} scrored ${sorted_mark[roll_no.indexOf(p)]} marks`;
+      }
+      //take the roll no from user
+      var roll_no = [];
```

Comment:
Create Like this, Because Each roll Roll Number Belongs There own Marks
```suggestion
      var rollNoWithMarks = [];
```

Reply:
changes done